README.md

```diff
@@ -71,6 +71,14 @@ php bin/magento setup:di:compile
 php bin/magento setup:static-content:deploy
 ```
 
+## Contributing
+
+To develop using our `docker-compose` structure just run:
+
+```bash
+$ chmod +x $(pwd)/.scripts/install.sh && $(pwd)/.scripts/install.sh
```

Comment:
You could apply and commit this permission for this file.

Reply:
Actually, it's already been done. I just didn't realize it. I'll change the README to reflect it.